esda/moran.py

```diff
@@ -67,7 +67,7 @@ class Moran:
     transformation  : string
                       weights transformation,  default is row-standardized "r".
                       Other options include "B": binary,  "D":
-                      doubly-standardized,  "U": untransformed
+                      doubly-standardized,  "O": restore original transformation
```

Comment:
```suggestion
                      doubly-standardized,  "O": restore original transformation 
                      (applicable only if ``w`` is  passed as ``W``)
```

Could you apply the same below as well?

Reply:
Done!

Reply:
It seems you haven't pushed that commit?